### PR TITLE
Scripts and configurations to run Jitsi locally

### PR DESCRIPTION
**Pull request by anaulin**
This PR adds configuration files and scripts to build and run a local dockerized Jitsi Meet, and documentation about how to use this. This work was motivated by one of the items in https://github.com/zinc-collective/convene/issues/40, which will require us to work on configuring authentication of some sort between `convene-web` and Jitsi.

I've also taken the liberty of trying to improve the organization (and amount of emojis) in the main `README`. My main goal was to make that document a little bit more "scannable", and to make it a little clearer to new folks wanting to develop in this repo that the bulk of developer documentation is under `convene-web`. I'm not sure if I succeeded in those goals, and I'm curious to hear what y'all think.

### CODE REVIEW

**Comment by anaulin** [comment]
@user512 @zspencer please review 🙏

File: README.md [hunk 1]
```diff
@@ -103,13 +107,23 @@ At present, Convene is split into three modules:
 - `web` which provides the human interface for Convene.
 - `api` which stores data long-term and provides a programmatic interface.
```
Comment (anaulin) [inline README.md:108]: Out of curiosity, for my own education, @zspencer is this `api` served by the Rails server in `convene-web`? Or is this somewhere else?

Comment (zspencer) [reply README.md:108]: At this point, we have not started work on the API; but it makes sense (to me) to include it in the `convene-web` project.

File: infrastructure/dev/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Development infrastructure
+
+This directory contains the scaffolding to build and run a local Dockerized
+version of the Jitsi server chain. To build and run this Dockerized setup, a
+convenience script is provided at [`bin/jitsi-local-run`](../bin/jitsi-local-run).
+
+The "entry point" for this setup is the
+[`jitsi-meet.Dorckerfile`](jitsi-meet.Dockerfile), which specifies how to
+build a Docker image for a local Jitsi setup. This Dockerfile depends, in
+turn, on the shell scripts in this directory.
```
Comment (zspencer) [inline infrastructure/dev/README.md:10]: Not blocking: I like that we have sprouted a folder for encapsulating how to run jitsi meet on a local machine; but I wonder if the naming is slightly off. It's not so much `dev` as it is `jitsi-meet-local`.

File: infrastructure/dev/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Development infrastructure
+
+This directory contains the scaffolding to build and run a local Dockerized
+version of the Jitsi server chain. To build and run this Dockerized setup, a
+convenience script is provided at [`bin/jitsi-local-run`](../bin/jitsi-local-run).
+
+The "entry point" for this setup is the
+[`jitsi-meet.Dorckerfile`](jitsi-meet.Dockerfile), which specifies how to
```
Comment (zspencer) [inline infrastructure/dev/README.md:8]: I believe you mean `jitsi-meet.Dockerfile` and not `jitsi-meet.Dorkerfile` :D

**Review by zspencer — APPROVED** [review]
I love this! Thank you! I dropped some nitpicks that are more about "expressing the thinking I have been doing and sucking at communicating" and less "do this instead."

**Comment by user512** [comment]
I am not familiar with Docker and will need one of you to help me to get run this locally.

**Comment by anaulin** [comment]
Thanks for the review, @zspencer ! Not sure if the merge was intentional or reflexive, but I'll follow up with a second PR addressing some of the stuff in the comments (at the very least fix the typo!).

@user512 maybe we can help you during today's pairing session?

**Comment by KellyAH** [comment]
I'm super late to this conversation but I just ❤️ the readme improvements and dockerfile you added in this PR! Awesome job! It looks so much better. Love the emojis!

Tho I still find it confusing that the readme doesn't clearly explain what the zinc collective is and its relationship to convene and its other products. I'll try updating the readme with that info.

And I can now hopefully spin up the app in a dockerfile and not have install tons more app versions on my local. Thanks for so many awesome improvements!

**Comment by anaulin** [comment]
Aww, thanks for the kind words, @KellyAH !

**Comment by zspencer** [comment]
> Thanks for the review, @zspencer ! Not sure if the merge was intentional or reflexive, but I'll follow up with a second PR addressing some of the stuff in the comments (at the very least fix the typo!).

At present, we work from the "if it's safe to merge, merge." This may not be our long-term practice, but establishing a pattern of relatively-low-friction small changes feels like a powerful way to make contributing more accessible. That said, it could also make contributing feel less safe; so I'd be interested in hearing your thoughts on how it feels to you.

**Comment by anaulin** [comment]
> At present, we work from the "if it's safe to merge, merge." This may not be our long-term practice, but establishing a pattern of relatively-low-friction small changes feels like a powerful way to make contributing more accessible. That said, it could also make contributing feel less safe; so I'd be interested in hearing your thoughts on how it feels to you.

**Comment by anaulin** [comment]
> At present, we work from the "if it's safe to merge, merge." This may not be our long-term practice, but establishing a pattern of relatively-low-friction small changes feels like a powerful way to make contributing more accessible. That said, it could also make contributing feel less safe; so I'd be interested in hearing your thoughts on how it feels to you.

I suspected, but wasn't sure, that you were defaulting to "merge if safe", which I like. Thanks for clarifying. I wonder if it might make sense to be explicit about that somewhere, maybe in the Contributing guidelines?

I agree that it makes contributing feel more welcoming. To me this doesn't make it feel unsafe to contribute (I try to be careful about being very explicit if I think something would be unsafe to merge), but of course this is just one opinion.

**Comment by zspencer** [comment]
@anaulin - Good point re: Contributing guidelines. I could have sworn I had it in there, but I guess not. I've created https://github.com/zinc-collective/www.zinc.coop/issues/60 and maybe I'll even wind up getting to it someday!

**Pull request merged**